docs/admin/master-node-communication.md

```diff
@@ -83,3 +83,57 @@ cluster (connecting to the ssh server listening on port 22) and passes all
 traffic destined for a kubelet, node, pod, or service through the tunnel.
 This tunnel ensures that the traffic is not exposed outside of the private
 GCE network in which the cluster is running.
+
+### Kubelet TLS Bootstrap 
+
+The 1.4 release introduces an experimental API for requesting certificates from a cluster-level CA. The first supported use of this API is the provisioning of TLS client certificates for kubelets. The proposal can be found [here] and progress on the feature is being tracked as [feature #43](https://github.com/kubernetes/features/issues/43). 
+
+Workflow 
+
+##### apiserver configuration
+A token file specifies at least one token as a "bootstrap token" assigned to a kubelet boostrap-specific group. This group will later be used in the controller-manager configuration to scope approvals in the default approval controller. As this feature matures, this token should also be bound to an RBAC policy that limits requests using the bootstrap token to only be able to make requests related to certificate provisioning. When RBAC policy is in place, scoping the tokens to a group will allow great flexibility (e.g. you could disable a particular bootstrap group's access when you are done provisioning the nodes). 
```

Comment:
Avoid passive voice: "This token should also..." becomes "You should ensure your token is bound to an RBAC policy..." This is to make it clear whose job it is to do things.